editions/server-external-js/tiddlywiki.info

```diff
@@ -10,17 +10,21 @@
 		"tiddlywiki/snowwhite"
 	],
 	"build": {
-		"listen": [
-			"--listen","root-tiddler=$:/core/save/all-external-js"],
 		"index": [
 			"--rendertiddler","$:/core/save/offline-external-js","index.html","text/plain",
-			"--render","$:/core/templates/tiddlywiki5.js","[[twcore-]addsuffix<version>addsuffix[.js]]","text/plain"],
+			"--render","$:/core/templates/tiddlywiki5.js","[[twcore-]addsuffix<version>addsuffix[.js]]","text/plain"
+			],
 		"static": [
 			"--rendertiddler","$:/core/templates/static.template.html","static.html","text/plain",
 			"--rendertiddler","$:/core/templates/alltiddlers.template.html","alltiddlers.html","text/plain",
 			"--rendertiddlers","[!is[system]]","$:/core/templates/static.tiddler.html","static","text/plain",
-			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"],
+			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"
+			],
 		"twcore": [
-			"--render","$:/core/templates/tiddlywiki5.js","[[twcore-]addsuffix<version>addsuffix[.js]]","text/plain"]
+			"--render","$:/core/templates/tiddlywiki5.js","[[twcore-]addsuffix<version>addsuffix[.js]]","text/plain"
+			],
+		"listen": [
+			"--listen","root-tiddler=$:/core/save/all-external-js"
+			]
```

Comment:
@cdruan ... This options should be moved to the end of the list in your upstream PR. 

If you do `tiddlywiki .xxxx --build`  it will start at the top and build all elements. .. If this option is the first one it will stop there. The rest will probably be built after the "listen" process stopped. .. I think this is a bit "wonky"

Reply:
Wo! I didn't know about the missing target behavior of the --build command. Thanks, @pmario!

I will move "listen" target to the end of the list.

Reply:
wait... I got curious and just tested what would happen if "listen" was listed first. It looks like tw still builds all of the targets before starting to listen. So I guess no change is needed.